deploy/helm/sumologic/values.yaml

```diff
@@ -5108,10 +5108,14 @@ tailing-sidecar-operator:
 ## ref: https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator
 opentelemetry-operator:
   enabled: false
-  createDefaultInstrumentation: true
+
+  ## Specific for Tracing - Instrumentation resource creation
+  createDefaultInstrumentation: false
+  instrumentationNamespaces: ""
+
   admissionWebhooks:
     failurePolicy: Fail
-    enabled: false
+    enabled: true
```

Comment:
Isn't this a breaking change?

Reply:
It's not as in version before admissionWebhooks were created by us with certificates.